src/webextension/background/message-ports.js

```diff
@@ -27,46 +27,85 @@ export default function initializeMessagePorts() {
 
   browser.runtime.onMessage.addListener(async (message, sender) => {
     switch (message.type) {
+    case "account_status":
+      return {account: getAccount().status()};
```

Comment:
Maybe call this message type `"get_account_details"` or `"get_account"`? As above, "status" was confusing me, and adding "get" helps with consistency, since all the other message types are verbs.